jenkins-scripts/dsl/brew_release.dsl

```diff
@@ -111,40 +111,37 @@ release_job.with
        pattern("${PR_URL_export_file_name}")
        allowEmpty()
      }
-
-     downstreamParameterized
-     {
-        trigger(bottle_builder_job_name)
-        {
-          condition('SUCCESS')
-          parameters {
-            currentBuild()
-            propertiesFile("${PR_URL_export_file_name}")
-          }
-        }
-     }
    }
 }
 
 // -------------------------------------------------------------------
 // 2. BREW bottle creation MATRIX job from pullrequest
 def bottle_job_builder = matrixJob(bottle_builder_job_name)
-OSRFOsXBase.create(bottle_job_builder)
+// set enable_github_pr_integration flag to false so we can customize trigger behavior
+OSRFBrewCompilationAnyGitHub.create(bottle_job_builder,
+                                    "osrf/homebrew-simulation",
+                                    false, [], false)
 GenericRemoteToken.create(bottle_job_builder)
 
-include_common_params(bottle_job_builder)
 bottle_job_builder.with
 {
    wrappers {
         preBuildCleanup()
+        credentialsBinding {
+          string('GITHUB_TOKEN', '6f03ada6-fae8-4e74-9e2b-d6d0cf4b97a2')
+        }
+   }
+
+   properties {
+     priority 100
```

Comment:
top priority, I agree (in line with debbuilders), just double checking that this was your intention.

Reply:
yes, this was my intention; I added it explicitly since I removed the `include_common_params` line from this job